android/java/res/layout/fragment_connect_account.xml

```diff
@@ -36,13 +36,27 @@
         android:layout_height="wrap_content"
         android:orientation="horizontal">
 
-        <ImageView
-            android:id="@+id/favicon"
-            android:layout_height="120dp"
-            android:layout_width="120dp"
-            android:padding="16dp"
-            android:contentDescription="@null"
-            android:visibility="gone"/>
+        <com.google.android.material.card.MaterialCardView
+            android:id="@+id/image_container_card"
+            android:layout_width="60dp"
+            android:layout_height="60dp"
+            android:layout_margin="10dp"
+            android:elevation="2dp"
```

Comment:
nit: desktop and fignma doesn't have border and elevation so make sure to cross check with @jamesmudgett

Reply:
not relevant anymore, https://bravesoftware.slack.com/archives/C03H854K19V/p1654526116208399?thread_ts=1654181154.094749&cid=C03H854K19V